main.gms

```diff
@@ -588,11 +589,15 @@ $setGlobal cm_conoptv  conopt3    !! def = conopt3
 $setGlobal cm_ccsfosall  off        !! def = off
 
 $setGlobal cm_APscen  SSP2          !! def = SSP2
-$setGlobal cm_magicc_calibrateTemperature2000  uncalibrated  !! def=uncalibrated
-$setGlobal cm_magicc_config  OLDDEFAULT    !! def = OLDDEFAULT
-$setGlobal cm_magicc_temperatureImpulseResponse  off           !! def = off
+$setGlobal cm_magicc_calibrateTemperature2000  HADCRUT4  !! def=uncalibrated
+$setGlobal cm_magicc_config  RCP26_50    !! def = OLDDEFAULT
+$setGlobal cm_magicc_temperatureImpulseResponse  on           !! def = off
 
 $setGlobal cm_damage_DiceLike_specification  HowardNonCatastrophic   !! def = HowardNonCatastrophic
+$setGlobal cm_damage_Labor_exposure  low    !!def = low
+$setGlobal cm_TCssp  SSP2  !! def = SSP2
```

Comment:
and these should stay

Reply:
Exactly, because of these two changes main.gms is part of the commit.

Reply:
I always use `git add -p main.gms` which lets you select with entering `y` and `n` which part of main.gms should be added to the commit.